transforms/universal/fdedup/utils/Makefile.local

```diff
@@ -0,0 +1,18 @@
+PYTHON=python
+PIP=pip
+
+venv:	requirements.txt
+	$(PYTHON) -m venv venv
+	if [ -e venv/Scripts/activate ]; then			\
+		echo "For Windows please try the following AS Administrator - no guarantees";	\
+		echo "  venv\\Scripts\\activate";		\
+		echo "  pip install --upgrade pip";		\
+		echo "  pip install -r requirements.txt";	\
+		echo "  pip install pytest";		\
+	else						\
+		. venv/bin/activate;			\
+		$(PIP) install --upgrade pip;		\
+		$(PIP) install -r requirements.txt;	\
+	fi					
+set-versions:
```

Comment:
I guess you don't need this if you've renamed the file.  If you go back to Makefile, you will also need to add

clean test build publish